src/main/java/com/epam/wl/dao/BookOrderDAO.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.wl.dao;
+
+import com.epam.wl.dao.book_order_handlers.BookOrderListHandler;
+import com.epam.wl.dao.book_order_handlers.BookOrderOneHandler;
+import com.epam.wl.entities.BookInstance;
+import com.epam.wl.entities.BookOrder;
+import com.epam.wl.entities.UserOrder;
+import com.epam.wl.enums.BookOption;
+import com.epam.wl.executor.Executor;
+import com.epam.wl.executor.ResultHandler;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+
+public class BookOrderDAO {
+    private Executor executor;
+    private final ResultHandler<Optional<BookOrder>> bookOrderOneHandler = new BookOrderOneHandler();
+    private final ResultHandler<List<BookOrder>> bookOrderListHandler = new BookOrderListHandler();
+
+    public BookOrderDAO(DataSource dataSource) {
+        executor = new Executor(dataSource);
+    }
+
+    /**
+     * Create row in book_order table
+     * @param bookInstance
+     * @param userOrder
+     * @param bookOption
+     * @return 1 (number of rows changed) if success, or throws SQLexception
+     * @throws SQLException
+     */
+    @SuppressWarnings("JavaDoc")
+    int create(
+            final BookInstance bookInstance,
+            final UserOrder userOrder,
+            final BookOption bookOption)
+            throws SQLException {
+        String updateQuery = "INSERT INTO book_order (book_instanceid, user_orderid, option) VALUES(%d, %d, '%s');";
```

Comment:
Make all strings, that you use like queries, private static final, it'll be better for performance

Reply:
Done